config/prod/sara-working-machine.yaml

```diff
@@ -1,6 +1,6 @@
 # Provision an EC2 instance
 template_path: remote/managed-ec2-ubuntu-v1.yaml
-stack_name: tcga-test
+stack_name: sara-working-machine
```

Comment:
changing the stack name implies that you want to replace `tcga-test` with `sara-working-machine`, is this correct?  if so please create 2 separate PRs.  One for a new instance (sara-working-machine) and one to delete existing instance (tcga-test).

Reply:
that file is gonef rom y fork. i have no idea why it's still here.